IDEOorg/steps
# #161271964 Prevent duplicate phone number use

#### What does this PR do?
Add a constraint to the migrations to prevent users from using a phone number that has already been used. 

#### Description of Task to be completed?
- migration to be added to the database to include a constraint **UNIQUE** in the phone numbers column

#### How should this be manually tested?
- Pull this branch `bg-remove-duplicate-client-phone-number-161271964`
- Run `npm run start` on your terminal
- Run `sqitch verify` on the terminal 
- You will see the information about the migration on the terminal.

#### What are the relevant pivotal tracker stories?
[#161271964](https://www.pivotaltracker.com/n/projects/2204131/stories/161271964)
